### ISSUE TITLE
Accept email attachments as well?

### ISSUE THREAD

**Issue by missytake** [deltachat-bot/media-archive-bot#2]
Right now it seems that normal email attachments are ignored, and only Delta Chat messages are accepted. A consequence from the default settings of only showing Delta Chat messages?

![Screenshot_2020-04-07_14-45-43](https://user-images.githubusercontent.com/57442838/78670751-89e85900-78de-11ea-9f8b-59535352fd4a.png)

Even if you 
1. first send an image to the bot via Delta, and
2. then send an image to the bot via Thunderbird, but with the same email address,

the first one is received and put on the page, but the second one does not generate console output and does not appear on the page at all.

Do we want to be able to accept images via email, too?

**Comment by pabzm** [comment]
I don't think that is worth the effort currently. We don't have a real world use case for this bot. And it is primarily a Delta Chat bot, not an Email bot.

Let's rather spend time on the Feed/ActivityPub-thing?
